File: get_feature_335.py
import sqlite3
import json

conn = sqlite3.connect('assistant.db')
cursor = conn.cursor()
cursor.execute('SELECT id, category, name, description, steps, passes, in_progress FROM features WHERE id = 335')
row = cursor.fetchone()

if row:
    feature = {
        'id': row[0],
        'category': row[1],
        'name': row[2],
        'description': row[3],
        'steps': row[4],
        'passes': row[5],
        'in_progress': row[6]
    }
    print(json.dumps(feature, indent=2))
else:
    print("Feature 335 not found")

conn.close()

File: get_feature_335.js
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient();

async function getFeature() {
  try {
    const feature = await prisma.features.findUnique({
      where: { id: 335 }
    });
    console.log(JSON.stringify(feature, null, 2));
  } catch (error) {
    console.error('Error:', error.message);
  } finally {
    await prisma.$disconnect();
  }
}

getFeature();
